config/config.go

```diff
@@ -149,6 +150,14 @@ func (c *DbConfig) Validate() error {
 		return errors.New("db: configuration cannot be empty")
 	}
 
+	if c.PreparedDb && c.PreparedDbFilePath == "" {
+		return errors.New("headers import: prepared database file path cannot be empty when using prepared database is enabled")
+	}
+
+	if c.PreparedDb && !fileExists(c.PreparedDbFilePath) {
+		return fmt.Errorf("headers import: prepared database file does not exist at path %s", c.PreparedDbFilePath)
+	}
```

Comment:
Consider putting all `PreparedDb`-related check into one block. Just a suggestion.

```suggestion
	if c.PreparedDb {
		if c.PreparedDbFilePath == "" {
			return errors.New("headers import: prepared database file path cannot be empty when using prepared database is enabled")
		}
		if !fileExists(c.PreparedDbFilePath) {
			return fmt.Errorf("headers import: prepared database file does not exist at path %s", c.PreparedDbFilePath)
		}
	}
```

Reply:
done ✅